extensions/resource-deployment/src/typings/ref.d.ts

```diff
@@ -6,4 +6,8 @@
 /// <reference path='../../../../src/vs/vscode.d.ts'/>
 /// <reference path='../../../../src/sql/azdata.d.ts'/>
 /// <reference path='../../../../src/sql/azdata.proposed.d.ts'/>
-/// <reference types='@types/node'/>
\ No newline at end of file
+/// <reference types='@types/node'/>
+declare module "sudo-prompt" {
+
+	export function exec(cmd: string, options: { name?: string, icns?: string }, callback: (error: string, stdout: string, stderr: string) => void): any;
+}
```

Comment:
We already have this in [https://github.com/microsoft/azuredatastudio/blob/master/src/typings/sudo-prompt.d.ts](https://github.com/microsoft/azuredatastudio/blob/master/src/typings/sudo-prompt.d.ts)

Reply:
Thanks - changed to reference to that file and added my SudoOptions type definition to it.

Reply:
This is still here...in case it wasn't clear you should be able to add this line instead :

`/// <reference path='../../../../src/typings/sudo-prompt.d.ts'/>`


---
In reply to: [334575556](https://github.com/microsoft/azuredatastudio/pull/7647#discussion_r334575556) [](ancestors = 334575556)